src/SumoCoders/FrameworkCoreBundle/Resources/assets/coffee/framework.flash.coffee

```diff
@@ -0,0 +1,22 @@
+class Flash
+
+  add: (message, type) ->
+    alertId = Math.floor(Date.now() / 1000);
```

Comment:
waarom / 1000? het kan toch geen kwaad als da in milliseconde is? Integendeel: de kans op duplicate id's is kleiner!